src/screens/Home/index.tsx

```diff
@@ -1,15 +1,23 @@
-import React, {useState} from 'react';
-import {Image, TouchableOpacity} from 'react-native';
+import React, {useState, useEffect, useCallback} from 'react';
+import {
+    ActivityIndicator,
+    FlatList,
+    TouchableOpacity,
+    View,
+} from 'react-native';
 import headerLogo from '../../assets/images/headerLogo.png';
 import HeaderOptions from '../../components/HeaderOptions';
 import Icon from '../../components/Icon';
 import Separator from '../../components/Separator';
 import Text from '../../components/Text';
 import Row from '../../components/Row';
-import RowBetween from '../../components/RowBetween';
 import {Container, HeaderImage, OptionButton} from './styles';
 import CharacterCard from '../../components/CharacterCard';
-import {mockedData as item} from './mock';
+import {useNavigation} from '@react-navigation/native';
+import {ResponseItemProps, ResponseProps} from './types';
+import axios from 'axios';
```

Comment:
We are using the request from the `api.ts` service file... remove this axios import

Reply:
resolved in commit https://github.com/IagoWag/rickandmorty/commit/a71461893b6fea906309d9e3507e0c49e0d139f5